frontend/components/molecules/MProfilePhoto.tsx

```diff
@@ -0,0 +1,26 @@
+import { Grid, Box, Button } from '@material-ui/core';
+import Image from 'next/image';
+import profilePicture from '../../public/profilepicture.svg';
+
+const ProfilePhoto = () => {
+  return (
+    <Grid container direction="row" wrap="nowrap">
+      <Grid>
+        <Image src={profilePicture} alt="profile picture" />
```

Comment:
Update the src to either use the avatar from the backend or to use dicebear's randomly generated profile picture based on their username.

src={avatar || `https://avatars.dicebear.com/api/bottts/${username}.svg`}